agent/csi/mod_csi_docker.go

```diff
@@ -386,14 +594,40 @@ func (d *dockerBridge) List() (*dvol.ListResponse, error) {
 			continue
 		}
 
+		if _, ok := volMap[name]; ok {
+			d.ctx.WithField("volume", vi).Warn(
+				"docker-csi-bridge: List: skipped volume already in cache")
+			continue
+		}
+
+		d.ctx.WithField("volume", vi.Id.Values).WithField(
+			"name", name).Debug(
+			"docker-csi-bridge: List: found new volume")
```

Comment:
Do want to log every volume at this point?

Reply:
I had it it here before, but it was "new that wasn't in the cache", but that is completely wrong at this point.

Reply:
Hi @codenrhoden,

Well, at debug level it could be useful. What you're really asking, and I agree, is if there is some way we can finally introduce logging domains to RR. Yes. I mean, "Yes, we have to figure out how to do that."

Reply:
I still think this bit can go.